conf/crukmi.config

```diff
@@ -0,0 +1,57 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'Cancer Research UK Manchester Institute HPC cluster profile provided by nf-core/configs'
+  config_profile_contact = 'Stephen Kitcatt, Simon Pearce (@skitcattCRUKMI, @sppearce)'
+  config_profile_url = 'http://scicom.picr.man.ac.uk/projects/user-support/wiki'
+}
+
+env {
+    SINGULARITY_CACHEDIR = '/lmod/nextflow_software'
+}
+
+singularity {
+  enabled = true
+  autoMounts = true
+}
+
+process {
+  beforeScript = 'module load apps/singularity/3.8.0'
+  executor = 'pbs'
+
+  errorStrategy = {task.exitStatus in [143,137,104,134,139,140] ? 'retry' : 'finish'}
+  maxErrors = '-1'
+  maxRetries = 3
+
+  withLabel:process_low {
```

Comment:
These withName/withLabel should be moved to a pipeine-specific institutional config,

Reply:
this process_low can stay here, but the `withName: 'SAMTOOLS_MPILEUP` should indeed go into a pipeline specific config